frontend/package.json

```diff
@@ -1,94 +1,82 @@
 {
-  "name": "gatsby-starter-default",
+  "name": "cellxgene-data-portal",
   "private": true,
-  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "description": "The Corpora Data Portal (CDP) enables the publication, discovery and exploration of interoperable single-cell datasets. Data contributors can upload, review and publish datasets for private or public use. Via the portal, data consumers are able to discover, download and connect data to visualization tools such as cellxgene to perform further analysis. The goal of the CDP is to catalyze distributed collaboration of single-cell research by providing a large, well-labeled repository of interoperable datasets.",
   "version": "0.1.0",
-  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "author": "Chan Zuckerberg Initiative",
   "dependencies": {
-    "@blueprintjs/core": "^3.38.1",
+    "@blueprintjs/core": "^3.38.2",
     "@blueprintjs/icons": "^3.24.0",
-    "@loadable/component": "^5.14.0",
-    "babel-plugin-styled-components": "^1.11.1",
-    "bl": "^4.0.3",
-    "clipboard-copy": "^3.1.0",
-    "gatsby": "^2.31.1",
-    "gatsby-image": "^2.4.21",
-    "gatsby-plugin-manifest": "^2.4.35",
-    "gatsby-plugin-offline": "^3.2.35",
-    "gatsby-plugin-react-helmet": "^3.3.14",
-    "gatsby-plugin-sass": "^2.3.22",
-    "gatsby-plugin-sharp": "^2.6.43",
-    "gatsby-plugin-styled-components": "^3.3.14",
-    "gatsby-plugin-theme-ui": "^0.3.0",
-    "gatsby-plugin-typescript": "^2.4.23",
-    "gatsby-source-filesystem": "^2.3.35",
-    "gatsby-transformer-sharp": "^2.5.19",
-    "lodash-es": "^4.17.15",
+    "@loadable/component": "^5.14.1",
+    "babel-plugin-styled-components": "^1.12.0",
+    "bl": "^4.1.0",
+    "clipboard-copy": "^3.2.0",
+    "lodash": "^4.17.20",
+    "next": "^10.0.6",
+    "next-images": "^1.7.0",
+    "next-secure-headers": "^2.1.0",
     "node-fetch": "^2.6.1",
     "node-sass": "^4.14.1",
-    "prop-types": "^15.7.2",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
-    "react-helmet": "^5.2.1",
-    "react-query": "^2.25.2",
-    "styled-components": "^5.2.0",
-    "theme-ui": "^0.3.1"
+    "react-query": "^2.26.4",
+    "styled-components": "^5.2.1"
   },
   "devDependencies": {
+    "@babel/preset-react": "^7.12.13",
```

Comment:
these two babel plugins are needed for `jest` to work in `frontend/jest/jest-preprocess.js`